src/Jackalope/Security/AccessControlManager.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Jackalope\Security;
+use Jackalope\FactoryInterface;
+use Jackalope\ObjectManager;
+use Jackalope\Transport\AccessControlInterface;
+use PHPCR\RepositoryException;
+use PHPCR\Security\AccessControlException;
+use PHPCR\Security\AccessControlManagerInterface;
+use PHPCR\Security\AccessControlPolicyInterface;
+use PHPCR\Security\PrivilegeInterface;
+
+/**
+ * {@inheritDoc}
+ */
+class AccessControlManager implements AccessControlManagerInterface
+{
+    /**
+     * @var FactoryInterface
+     */
+    private $factory;
+
+    /**
+     * @var ObjectManager
+     */
+    private $om;
+
+    /**
+     * @var AccessControlInterface
+     */
+    private $transport;
+
+    /**
+     * @var PrivilegeInterface[]
+     */
+    private $supportedPrivilegesByPath = array();
+
+    public function __construct(FactoryInterface $factory, ObjectManager $om, AccessControlInterface $transport)
+    {
+        $this->factory = $factory;
+        $this->om = $om;
+        $this->transport = $transport;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getSupportedPrivileges($absPath = null)
+    {
+        if (!isset($this->supportedPrivilegesByPath[$absPath])) {
+            $this->supportedPrivilegesByPath[$absPath] = $this->transport->getSupportedPrivileges($absPath);
+        }
+
+        return $this->supportedPrivilegesByPath[$absPath];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function privilegeFromName($privilegeName)
+    {
+        foreach ($this->getSupportedPrivileges() as $privilege) {
+            if ($privilegeName === $privilege->getName()) {
+                return $privilege;
+            }
+
+            foreach ($privilege->getAggregatePrivileges() as $childPrivilege) {
+                if ($privilegeName === $childPrivilege->getName()) {
+                    return $childPrivilege;
+                }
+            }
+        }
+
+        throw new AccessControlException($privilegeName);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function hasPrivileges($absPath, array $privileges)
+    {
+        // TODO
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getPrivileges($absPath = null)
+    {
+        // TODO
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getPolicies($absPath)
+    {
+        return $this->om->getPolicies($absPath);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getEffectivePolicies($absPath)
+    {
+        throw new RepositoryException('This method can not properly be implemented. Use getPolicies for this path');
```

Comment:
yep, and angela thinks it makes semantically no sense, as you might not have the right to read all policies that might affect you, and there is no defined behaviour for that case. the things that are releavant are: 
* what policies are applicable at a specific path?
* do i have the permission to do something? this is SessionInterface::hasPermission